pyvista/core/utilities/geometric_objects.py

```diff
@@ -80,11 +85,24 @@ def Cylinder(
     Examples
     --------
     >>> import pyvista
-    >>> import numpy as np
     >>> cylinder = pyvista.Cylinder(
     ...     center=[1, 2, 3], direction=[1, 1, 1], radius=1, height=2
     ... )
     >>> cylinder.plot(show_edges=True, line_width=5, cpos='xy')
+
+    >>> import pyvista
+    >>> pl = pyvista.Plotter()
```

Comment:
This example is mostly about the similarity of `DataSet.plot` vs. `Plotter.add_mesh` rather than `Cylinder`.  I think it is fine to keep, but also fine to delete.

Reply:
I think we should keep it just for clarity